Fluid/FluidParser.cs

```diff
@@ -116,14 +112,23 @@ public FluidParser(FluidParserOptions parserOptions)
                 .AndSkip(RParen)
                 .Then(x => new RangeExpression(x.Item1, x.Item2));
 
-            // primary => NUMBER | STRING | BOOLEAN | property
+            // primary => NUMBER | STRING | property
             Primary.Parser =
                 String.Then<Expression>(x => new LiteralExpression(StringValue.Create(x)))
-                .Or(True.Then<Expression>(x => new LiteralExpression(BooleanValue.True)))
-                .Or(False.Then<Expression>(x => new LiteralExpression(BooleanValue.False)))
-                .Or(Empty.Then<Expression>(x => new LiteralExpression(EmptyValue.Instance)))
-                .Or(Blank.Then<Expression>(x => new LiteralExpression(BlankValue.Instance)))
-                .Or(Member.Then<Expression>(x => x))
+                .Or(Member.Then<Expression>(static x => {
+                    if (x.Segments.Count == 1)
+                    {
+                        switch ((x.Segments[0] as IdentifierSegment).Identifier)
+                        {
+                            case "empty": return new LiteralExpression(EmptyValue.Instance);
```

Comment:
would it make sense to have `internal static` field, `return LiteralExpression.EmptyValue`?

Reply:
Yes, these are immutable.